src/profiles.js

```diff
@@ -132,3 +130,191 @@ export function hookupProfile(profileMap, author) {
 
   return [profile, setProfile];
 }
+
+export function patchProfileBanners() {
```

Comment:
patchProfileBanners function belongs in its own file like profilePopout.js or something. profiles.js should just have the code for creating and managing the `profile` object.